CHANGES/5904.bugfix

```diff
@@ -0,0 +1 @@
+Dealing with Packages from S3
```

Comment:
Can this be phrased in terms of the error the user would have experienced? I'm not sure what error that is so I'm not sure what to suggest exactly.

Reply:
`NotImplementedError("This backend doesn't support absolute paths.")`
when serializer validates a `Package`, I have no idea how to phrase this